File: sem3/java/arm.java
import java.io.*;
class arm{

public static void main(String args[]){
	int a,r=0,s,f=0,b;
	
	a = Integer.parseInt(args[0]);
	b = a;
	
	for(int i=1;i<=3;i++){
		r = a%10;
		f = f+ r*r*r;
		a = a/10;
		
	}

if(f==b)
System.out.println(b + " is an armstrong number.");
else
System.out.println(b + " is not an armstrong number.");

	

}


}

File: sem2/c/arm.c
#include <stdio.h>
#include <math.h>

int main()
{
	int arm, sum=0, nd=0 , i,r , a;

	printf("Enter a number : \n");
	scanf("%d" , &arm);
	a = arm;

	if(a<0)
	return 0;

	while(a!=0){
		a = a/10;
		nd++;	
	}

	a = arm;
	
	for(i=0;i<nd;i++){

		r = a%10;
		sum = sum + pow(r, nd);
		a = a/10;
	
	}


	if(sum== arm){
	printf("%d is an armstrong number." , arm);
	}else{
	printf("%d is not an armstrong number." , arm);
	}
	printf("\n");
	return 0;


}
